mne/viz/tests/test_ica.py

```diff
@@ -72,6 +73,51 @@ def test_plot_ica_components():
 
 
 @requires_sklearn
+def test_plot_ica_properties():
+    """Test plotting of ICA properties
+    """
+    import matplotlib.pyplot as plt
+    raw = _get_raw()
+    picks = _get_picks(raw)
+    epochs = _get_epochs()
+    ica = ICA(noise_cov=read_cov(cov_fname), n_components=2,
+              max_pca_components=3, n_pca_components=3)
+    with warnings.catch_warnings(record=True):  # bad proj
+        ica.fit(raw, picks=picks)
+
+    # test _create_properties_layout
+    fig, ax = _create_properties_layout()
+    assert_equal(len(ax), 5)
+
+    ica.plot_properties(raw, picks=0)
+    ica.plot_properties(epochs, picks=1)
+    ica.plot_properties(epochs, picks=1, dB=True)
+    ica.plot_properties(epochs, picks=1, dB=True, cmap='jet')
+    ica.plot_properties(epochs, picks=1, image_kws={'sigma': 1.5})
+    ica.plot_properties(epochs, picks=1, topo_kws={'res': 10})
+    ica.plot_properties(epochs, picks=1, plot_std=False)
+    ica.plot_properties(epochs, picks=1, plot_std=1.5)
+    ica.plot_properties(epochs, picks=1, psd_kws={'fmax': 65.})
+    ica.plot_properties(epochs, picks=0, sigma=1.5)
+    plt.close('all')
+
+    assert_raises(ValueError, ica.plot_properties, epochs, dB=list('abc'))
+    assert_raises(ValueError, ica.plot_properties, epochs, plot_std=[])
+    assert_raises(ValueError, ica.plot_properties, ica)
+    assert_raises(ValueError, ica.plot_properties, [0.2])
+    assert_raises(ValueError, plot_properties, epochs, ica=epochs)
+
+    fig, ax = plt.subplots(2, 3)
+    ax = ax.ravel()[:-1]
+    ica.plot_properties(epochs, picks=1, axes=ax)
+    fig = ica.plot_properties(raw, picks=[0, 1])
+    assert_equal(len(fig), 2)
+    assert_raises(ValueError, plot_properties, epochs, ica=ica, picks=[0, 1],
+                  axes=ax)
+    plt.close('all')
```

Comment:
Could we save time by reducing direct calls to ica.plot_topomap that are also implicitly done here?